.gitignore

```diff
@@ -1,4 +1,5 @@
 obj/*
+imdb_data
 .DS_STORE
```

Comment:
(random line)
@mrks are you fine with join_order_benchmark.cpp being essentially a copy plus some additions of file_based_benchmark? Should I try to generalize?

Reply:
For now, that should be fine.